src/admin/mod.rs

```diff
@@ -121,9 +123,13 @@ impl Server {
                 }
             }));
 
+        let shutdown_trigger = shutdown.trigger();
         tokio::spawn(async move {
             info!("Serving admin server at {}", self.addr);
-            server.await
+            if let Err(err) = server.await {
+                eprintln!("Serving admin start failed: {err}");
```

Comment:
we should never use println, use `error!`

Reply:
okay, keep the same by using `error!`